app/controllers/restaurants_controller.rb

```diff
@@ -1,6 +1,11 @@
 class RestaurantsController < ApplicationController
   def index
-    @restaurants = Restaurant.all
+    if params[:search]
+      @assortment = params[:search]
+      @results = Restaurant.all.select { |restaurant| restaurant.assortment.include? @assortment.capitalize }
```

Comment:
when a line spans over 80 chars maybe consider a `do ... end` block instead of the one-liner equivalent

Reply:
not a blocker though...